api/Document.json

```
@@ -245,14 +245,28 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/alinkColor",
           "support": {
-            "chrome": {
-              "version_added": "64",
-              "notes": "Before Chrome 64, this property was accessed through the <code>HTMLDocument</code> alias."
-            },
-            "chrome_android": {
-              "version_added": "64",
-              "notes": "Before Chrome 64, this property was accessed through the <code>HTMLDocument</code> alias."
-            },
+            "chrome": [
+              {
+                "version_added": "64"
+              },
+              {
+                "version_added": "1",
+                "version_removed": "64",
+                "partial_implementation": true,
+                "notes": "Before Chrome 64, this property was accessed through the <code>HTMLDocument</code> alias."
```

```suggestion
                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/HTMLDocument'><code>HTMLDocument</code></a>, not all <code>Document</code> objects."
```

This is in order to resolve https://github.com/mdn/browser-compat-data/issues/6839, and I've used the wording for api.Node.contains from https://github.com/mdn/browser-compat-data/pull/6850.

I'd like @ddbeck to review this situation though, since it'll be a lot more visible than that single entry.

---

Yes, I prefer this new note text. I'm happy to see it applied elsewhere. 👍

---

Sounds good, I'll go ahead and apply the note text then!